src/kokkos_ext/ArborX_DetailsKokkosExtClassLambda.hpp

```diff
@@ -0,0 +1,26 @@
+/****************************************************************************
+ * Copyright (c) 2017-2023 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_DETAILS_KOKKOS_EXT_CLASS_LAMBDA
+#define ARBORX_DETAILS_KOKKOS_EXT_CLASS_LAMBDA
+
+#include <Kokkos_Macros.hpp>
+
+#if defined(KOKKOS_VERSION_GREATER_EQUAL) &&                                   \
```

Comment:
We define `KOKKOS_VERSION` in ArborX, so this could simply be
```c++
#if KOKKOS_VERSION >= 40000
```

Reply:
Yes, but I prefer using the macros for clarity.